smart-contracts/smart_contracts_controller.md

```diff
@@ -21,6 +21,7 @@ class SmartContractsController:
         gas_limit: uint32;
         guardian: Optional[Address];
         relayer: Optional[Address];
+        gas_price: Optional[uint32];
```

Comment:
Since here we are in specs, let's resort the arguments, and have `gas_limit` near `gas_price`.

Reply:
Here and below.